### DISCUSSION TITLE
How can I assign different ip addresses to different projects？

### DISCUSSION THREAD

**Discussion by ClarkPgOne** [vercel/community#6329] [Feature Question]
### Question

How do I assign different ip addresses to different projects?
<img width="1484" alt="image" src="https://github.com/vercel/community/assets/48192529/d2795604-1c82-44cc-a14c-d205f472a41f">

**Comment by swarnava** [comment]
Hey there,

You don't need separate IP to point to Vercel. As long as you have added correct domain per project, the single anycast IP should work for all project domains.
